luci/sqm-cbi.lua

```diff
@@ -57,11 +57,21 @@ function e.write(self, section, value)
 end
 -- TODO: inform the user what we just did...
 
+
+-- Add to physical interface list a hint of the correpsonding network names,
+-- used to help users better select e.g. lan or wan interface.
+
 n = s:taboption("tab_basic", ListValue, "interface", translate("Interface name"))
 -- sm lifted from luci-app-wol, the original implementation failed to show pppoe-ge00 type interface names
 for _, iface in ipairs(ifaces) do
-	if not (iface == "lo" or iface:match("^ifb.*")) then
-		n:value(iface)
+	local nets = net:get_interface(iface)
+	if nets and not (iface == "lo" or iface:match("^ifb.*")) then
```

Comment:
Mea culpa -- pushed the wrong commit but now corrected. Thanks for the patience.